jax/scipy/optimize/line_search.py

```diff
@@ -0,0 +1,121 @@
+import jax.numpy as np
+import jax
+from jax.lax import while_loop, cond
+from collections import namedtuple
+
+LineSearchResults = namedtuple('LineSearchResults', ['failed', 'nfev', 'ngev', 'k', 'a_k', 'f_k', 'g_k'])
+
+
+def line_search_nojit(value_and_gradient, position, direction, f_0=None, g_0=None, max_iterations=50, c1=1e-4, c2=0.9):
```

Comment:
Please move or remove this, same as for `minimize_nojit` :)

Reply:
Done.